__tests__/app.test.js

```diff
@@ -88,6 +88,54 @@ describe('GET /api/articles', () => {
               });
         })
     })
+    test('GET: 200 should also accept the following query: topic(mitch), which filters the articles by the topic value specified in the query.', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles.length).toBe(12);
+            body.articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: 'mitch',
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(String)
+                })
+            })
+        })
+    })
```

Comment:
Great happy path test 👍 
I like that you were as specific as possible in your `toMatchObject` matcher and specified the topic 🙌